src/loginServices.js

```diff
@@ -113,21 +116,32 @@ export function loadTranslations(store, locale, defaultTranslations = {}) {
 
   // Update dir- and lang-attributes on the HTML element
   // when the locale changes
-  document.documentElement.setAttribute('lang', parentLocale);
-  document.documentElement.setAttribute('dir', rtlDetect.getLangDir(locale));
+  document.documentElement.setAttribute('lang', locale);
+  document.documentElement.setAttribute('dir', rtlDetect.getLangDir(parentLocale));
 
   // Set locale for Moment.js (en is not importable as it is not stored separately)
   if (parentLocale === 'en') moment.locale(parentLocale);
   else {
-    import(`moment/locale/${parentLocale}`).then(() => {
-      moment.locale(parentLocale);
-    }).catch(e => {
-      // eslint-disable-next-line no-console
-      console.error(`Error loading locale ${parentLocale} for Moment.js`, e);
+    // For moment, we want to import and load the most-specific
+    // locale possible without the numbering system suffix,
+    // e.g. it-IT if available, falling back to it if that fails.
+    import(`moment/locale/${locale}`).then(() => {
+      moment.locale(locale);
+    }).catch(() => {
+      import(`moment/locale/${parentLocale}`).then(() => {
+        moment.locale(parentLocale);
+      }).catch(e => {
+        // eslint-disable-next-line no-console
+        console.error(`Error loading locale ${parentLocale} for Moment.js`, e);
```

Comment:
Given a locale like `en-SE-u-nu-latn`, this will run
```
import('en-SE-u-nu-latn')
.catch(() => import ('en'))
```
But what we want to be doing is
```
import('en-SE')
.catch(() => import ('en'))
```

Reply:
@zburke good point. Done!